ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## Upcoming Release
+#### Az.SelfHelp
+* Init cmdlets support for the api-version 2023-03-01-preview.
```

Comment:
This is the overall changelog for Az. Please revert your changes and leave the changelog as a comment.

Reply:
sure reverted the change